WordPress/Classes/ViewRelated/NUX/LoginFields.h

```diff
@@ -41,5 +41,7 @@ NS_ASSUME_NONNULL_BEGIN
                            userIsDotCom:(BOOL)userIsDotCom
                shouldDisplayMultiFactor:(BOOL)shouldDisplayMultifactor;
 
+- (NSDictionary *)helpshiftLoginOptions;
```

Comment:
Hmm... `sourceTag` might be a better fit in `HelpshiftPresenter` maybe?  But, agreed, something for the future.